frame/system/src/lib.rs

```diff
@@ -394,30 +393,40 @@ pub mod pallet {
 
 		/// Set the new runtime code.
 		///
-		/// ## Complexity
 		/// - `O(C + S)` where `C` length of `code` and `S` complexity of `can_set_code`
+		/// - 1 call to `can_set_code`: `O(S)` (calls `sp_io::misc::runtime_version` which is
+		///   expensive).
+		/// - 1 storage write (codec `O(C)`).
+		/// - 1 digest item.
+		/// - 1 event.
```

Comment:
I dont think having these explicitly written down helps much.  
We recently had a MR to clean most of them up, since they get easily out-dated and are less helpful than just looking at the weight file.